exporters/otlp/otlplog/otlploggrpc/config.go

```diff
@@ -36,92 +35,6 @@ type RetryConfig struct {
 	// TODO: implement.
 }
 
-// WithInsecure disables client transport security for the Exporter's gRPC
-// connection, just like grpc.WithInsecure()
-// (https://pkg.go.dev/google.golang.org/grpc#WithInsecure) does.
-//
-// If the OTEL_EXPORTER_OTLP_ENDPOINT or OTEL_EXPORTER_OTLP_LOGS_ENDPOINT
```

Comment:
We must remember to document all the env vars in `Expoter` type doc comments or package documentation.
This can be handled as a separate issue.